### PR TITLE
Add documentation to twilio webhooks

### PR DESCRIPTION
**Pull request by mabumusa1**
Explains the process of enabling webhooks from twilio to receive SMS and mark contacts as DNC

### CODE REVIEW

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md [hunk 1]
```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
 
 The new Text message will be pre-selected so you can save the *Send Text Message* action as well. You can use the action in your Campaign dripflow.
 
+## Tracking Replies and Unsubscribes
+
+Contacts can unsubscribe from your SMS messags by replying with the word "Stop" to your SMS, Once Mautic recive this SMS that specific contact will be marked as Do not Contact and will not be contacted again.
```
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:39]: A couple of typos, and I believe the DNC is set per-channel so we should be clear that they will not be contacted by SMS but may still continue to be contacted via other channels (that is my understanding of how it works, at least).

```suggestion
Contacts can unsubscribe from your SMS messages by replying with the word "Stop" to your SMS. Once Mautic receives this SMS, that specific contact will be marked as Do Not Contact (DNC) and will not be contacted by SMS again.
```

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md [hunk 2]
```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
...
+
+You can also get SMS replies in your contact's timeline like this 
+![](contact-reply.png)
```
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:42]:
```suggestion
![Screenshot of contact reply in their timeline](contact-reply.png)
```

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md [hunk 3]
```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
...
+
+## Configure Twilio Webhooks
+
+In order to make twilio send back the replies to Mautic you have to do the following steps:
```
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:46]:
```suggestion
In order to make Twilio send back the replies to Mautic, you have to take the following steps: 
```

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md [hunk 4]
```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
...
+
+Contacts can unsubscribe from your SMS messags by replying with the word "Stop" to your SMS, Once Mautic recive this SMS that specific contact will be marked as Do not Contact and will not be contacted again.
+
+You can also get SMS replies in your contact's timeline like this 
+![](contact-reply.png)
+
+## Configure Twilio Webhooks
+
+In order to make twilio send back the replies to Mautic you have to do the following steps: 
+1. Go to twilio dashboard -> Phone Numbers -> Manage Numbers -> Active Numbers, then select the number you are sending from (the same number you configured in the steps above). or simple follow this link [twilio-active-numbers]
```
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:47]:
```suggestion
1. Go to the Twilio dashboard -> Phone Numbers -> Manage Numbers -> Active Numbers, then select the number you are sending from (the same number you configured in the steps above) or follow [this link][twilio-active-numbers]
```

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md [hunk 5]
```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
...
+
+2. Go the messaging section, configure with webhook as shown in the screenshot below. make sure you point back to your server like this `https://domain.com/sms/twilio/callback`
```
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:49]:
```suggestion
2. Go the messaging section, configure with a webhook as shown in the screenshot below. Make sure you point back to your server like this `https://example.com/mautic/sms/twilio/callback` replacing example.com/mautic with the path to your Mautic instance.
```

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md [hunk 6]
```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
...
+![](twilio-webhook.png)
+
+After these configurations you twilio will notify Mautic with any incoming SMS
```
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:52]:
```suggestion
After these configurations, Twilio will notify Mautic of any incoming SMS messages in response to your campaign.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for adding this information to the docs @mabumusa1 - a few typos and some suggestions to consider :)

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md [hunk 7]
```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
...
+You can also get SMS replies in your contact's timeline like this 
+![](contact-reply.png)
+
+## Configure Twilio Webhooks
+
+In order to make twilio send back the replies to Mautic you have to do the following steps: 
+1. Go to twilio dashboard -> Phone Numbers -> Manage Numbers -> Active Numbers, then select the number you are sending from (the same number you configured in the steps above). or simple follow this link [twilio-active-numbers]
+
+2. Go the messaging section, configure with webhook as shown in the screenshot below. make sure you point back to your server like this `https://domain.com/sms/twilio/callback`
+![](twilio-webhook.png)
```
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:50]:
```suggestion
![Screenshot of Twilio webhook settings](twilio-webhook.png)
```

File: pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md (hunk 6 above)
Comment (RCheesley) [inline pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md:52]:
```suggestion
After these configurations, Twilio will notify Mautic of any incoming SMS messages in response to your campaign.```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Sorry I messed up one of the file comments 😊

**Review by mabumusa1 — COMMENTED** [review]
The changes look good to me, let's use them

**Review by RCheesley — APPROVED** [review]
LGTM! 🎉

**Comment by RCheesley** [comment]
Looks like the changes were not enacted with a commit so I have just manually made them in the UI.

**Pull request merged**